src/networks.js

```diff
@@ -10,15 +11,47 @@ export const RINKEBY_STAGING_COURT =
   '0x52180Af656A1923024D1ACcF1D827AB85cE48878'
 
 export const networks = {
-  rpc: { court: '0xD833215cBcc3f914bD1C9ece3EE7BF8B14f841bb' },
```

Comment:
A bit of a nitpick, but let's order these networks as:

1. Mainnet
2. Rinkeby
3. Ropsten
4. Local

This more or less reflects their relative importance to us as environments.